ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -1165,4 +1164,54 @@ extension SignalProducerType {
 	public func wait() -> Result<(), Error> {
 		return then(SignalProducer<(), Error>(value: ())).last() ?? .Success(())
 	}
+
+	/// Creates a new `SignalProducer` that will multicast values emitted by
+	/// the underlying producer, up to `capacity`.
+	///
+	/// The underlying `SignalProducer` will not be started until `self` is started
+	/// for the first time. When subscribing to this producer, all previous values
+	/// (up to `capacity`) will be emitted, followed by any new values.
+	///
+	/// If you find yourself needing *the current value* (the last buffered value)
+	/// you should consider using `PropertyType` instead, which, unlike this operator,
+	/// will guarantee at compile time that there's always a buffered value.
+	///
+	/// This operator has the same semantics as `SignalProducer.buffer`.
+	@warn_unused_result(message="Did you forget to call `start` on the producer?")
+	public func replayLazily(capacity: Int = Int.max) -> SignalProducer<Value, Error> {
```

Comment:
Is that a sensible default? With a "noisy" producer, this could inadvertently use way too much memory. Not sure I have a better suggestion though. I think using 1 may be a common case, but I don't think that that would pass the "principle of least surprise" test for the API...